packages/sacado/src/mpl/Sacado_mpl_is_convertible.hpp

```diff
@@ -32,56 +32,12 @@
 
 #include "Sacado_ConfigDefs.h"
 
-#ifdef HAVE_SACADO_CXX11
-
 #include <type_traits>
```

Comment:
@etphipp I guess the file can simply be deleted now :)

Reply:
True, but we would also need to make sure this isn't called anywhere